org.freac.freac.appdata.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2018 Robert Kausch -->
+<component type="desktop-application">
+    <id>org.freac.freac</id>
+    <launchable type="desktop-id">org.freac.freac.desktop</launchable>
+    <name>fre:ac</name>
+    <summary>Free and open source audio converter</summary>
+    <metadata_license>FSFAP</metadata_license>
+    <project_license>GPL-2.0</project_license>
+    <update_contact>support@freac.org</update_contact>
+    <content_rating type="oars-1.0">
+        <content_attribute id="money-purchasing">mild</content_attribute>
+    </content_rating>
+    <description>
+        <p>
+            fre:ac is a free and open source audio converter. It comes with
+            an integrated tag editor and converts between various formats.
+        </p>
+        <p>
+            Features include:
+        </p>
+        <ul>
+            <li>Support for MP3, MP4/AAC, Opus, FLAC and other formats</li>
+            <li>Integrated player and tag editor</li>
+            <li>Support for playlists, cue sheets and chapters</li>
+            <li>Fast, multi-threaded conversion engine</li>
+            <li>Ability to convert to multiple formats at once</li>
+            <li>Can verify lossless input and output files</li>
+            <li>Support for freedb/CDDB queries and submits</li>
+            <li>Support for using command line codecs</li>
+            <li>User interface available in 40+ languages</li>
+            <li>Optional command line interface</li>
+        </ul>
+    </description>
+    <url type="homepage">https://www.freac.org</url>
+    <url type="bugtracker">https://github.com/enzo1982/freac/issues</url>
+    <url type="donation">https://www.freac.org/donating.php</url>
+    <screenshots>
+        <screenshot type="default">
+            <image>https://www.freac.org/images/screenshots/freac-linux.png</image>
+        </screenshot>
+        <screenshot>
+            <image>https://www.freac.org/images/screenshots/freac-linux-tageditor.png</image>
+        </screenshot>
+        <screenshot>
+            <image>https://www.freac.org/images/screenshots/freac-linux-settings.png</image>
+        </screenshot>
+    </screenshots>
+    <releases>
+        <release version="1.1 alpha 20181201a" date="2018-12-02">
```

Comment:
You might encounter problems with this versioning later, they are used to compare versions.

Reply:
OK, I'll use `1.1~alpha20181201a`. That nicely sorts before even `1.1` and is what is already used on e.g. Haiku.